camel/toolkits/ask_news_toolkit.py

```diff
@@ -0,0 +1,105 @@
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =========== Copyright 2023 @ CAMEL-AI.org. All Rights Reserved. ===========
+import os
+from typing import List, Literal
+
+from asknews_sdk import AskNewsSDK
+
+from camel.toolkits.openai_function import OpenAIFunction
+
+
+class AskNewsToolkit:
+    r"""A class representing a toolkit for interacting with the AskNews API.
+
+    This class provides methods for fetching news, stories, and other content
+    based on user queries using the AskNews API.
+    """
+
+    def __init__(self):
+        r"""Initialize the AskNewsToolkit with API clients.
+
+        The API keys and credentials are retrieved from environment variables.
+        """
+        # Initialize the AskNews client using API keys and credentials from
+        # environment variables.
+        self.asknews_client = AskNewsSDK(
+            client_id=os.environ.get("ASKNEWS_CLIENT_ID"),
+            client_secret=os.environ.get("ASKNEWS_CLIENT_SECRET"),
+            scopes=["chat", "news", "stories"],
+        )
+
+    def get_news(
+        self,
+        query: str,
+        n_articles: int = 10,
+        return_type: Literal["string", "dict"] = "string",
```

Comment:
I think here should be `dicts`, we may also need to add `both`

```suggestion
        return_type: Literal["string", "dicts"] = "string",
```